core/tests/unittests/bayesopt/gpautograd/test_comparison_gpy.py

```diff
@@ -3,8 +3,8 @@
 import numpy as np
 import matplotlib.pyplot as plt
 import pickle
-import pytest
 import tempfile
+#import pytest
```

Comment:
Should we remove the comment?

Reply:
Yes, given the test works now, I'll remove this